ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/preexecution/PreExecutingValidatingCommitter.scala

```diff
@@ -102,13 +90,17 @@ class PreExecutingValidatingCommitter[StateValue, ReadSet, WriteSet](
             ),
           )
         } yield {
-          submissionAggregator.finish()
           submissionResult
-        }
+        }).transform {
+          case Failure(_: ConflictDetectedException) =>
+            logger.error("Too many conflicts detected during post-execution. Giving up.")
+            Success(SubmissionResult.Acknowledged) // But it will simply be dropped.
+          case result => result
```

Comment:
In the interest of not having catch-all cases:

```suggestion
          case result: Success => result
```